### PR TITLE
Allow Cardinality Plot to work with individual datasets

### PR DESCRIPTION
**Pull request by lajohn4747**
Part 3 of 3 for allowing visualizations to work with a single dataset: #581 
CU-86b0t56r9

This PR is to handle `get_cardinality_plot`

get_cardinality_bar_both
get_cardinality_bar_real
get_cardinality_bar_synth
get_cardinality_dist_both
get_cardinality_dist_real
get_cardinality_dist_synth

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86b0t56r9 SDMetrics - Allow me to visualize just the real or synthetic data #581](https://app.clickup.com/t/86b0t56r9)

**Review by frances-h — APPROVED** [review]

File: sdmetrics/visualization.py [hunk 1]
```diff
@@ -365,6 +365,28 @@ def _generate_column_plot(
...
+        raise ValueError('Cannot get max between two None values.')
+    if real_data is None:
+        return max(synthetic_data)
+    elif synthetic_data is None:
+        return max(real_data)
+    else:
+        return max(max(real_data), max(synthetic_data))
+
+
+def _get_min_between_datasets(real_data, synthetic_data):
```
Comment (amontanez24) [inline sdmetrics/visualization.py:379]: can we add unit tests for these methods or unit tests for `_generate_cardinality_plot` that will test these code paths

File: sdmetrics/visualization.py [hunk 2]
```diff
@@ -365,6 +365,28 @@ def _generate_column_plot(
...
 
+def _get_max_between_datasets(real_data, synthetic_data):
+    if synthetic_data is None and real_data is None:
+        raise ValueError('Cannot get max between two None values.')
+    if real_data is None:
+        return max(synthetic_data)
+    elif synthetic_data is None:
+        return max(real_data)
+    else:
+        return max(max(real_data), max(synthetic_data))
```
Comment (gsheni) [inline sdmetrics/visualization.py:376]:
```suggestion
    return max(max(real_data), max(synthetic_data))
```

File: sdmetrics/visualization.py [hunk 3]
```diff
@@ -365,6 +365,28 @@ def _generate_column_plot(
...
+
+def _get_min_between_datasets(real_data, synthetic_data):
+    if synthetic_data is None and real_data is None:
+        raise ValueError('Cannot get min between two None values.')
+    if real_data is None:
+        return min(synthetic_data)
+    elif synthetic_data is None:
+        return min(real_data)
+    else:
+        return min(min(real_data), min(synthetic_data))
```
Comment (gsheni) [inline sdmetrics/visualization.py:387]:
```suggestion
    return min(min(real_data), min(synthetic_data))
```

File: sdmetrics/visualization.py (hunk 1 above)
Comment (lajohn4747) [reply sdmetrics/visualization.py]: Added

**Pull request merged**